front_camera:
  ros__parameters:
    height: 720
    width: 1080

r_motor:
  ros__parameters:
    frequency: 59
    servo:
      ch: 0
      right: 7487
      center: 5887
      left: 4447
    dcmotor:
      ch: 1
      forward: 3183
      neutral_min: 6031
      neutral_max: 6175
      back: 8991

